src/r_texture_cvars.c

```diff
@@ -44,7 +44,7 @@ cvar_t gl_picmip = { "gl_picmip", "0", CVAR_RELOAD_GFX };
 cvar_t gl_miptexLevel = { "gl_miptexLevel", "0", CVAR_RELOAD_GFX, OnChange_gl_miptexLevel };
 cvar_t gl_texturemode = { "gl_texturemode", "GL_LINEAR_MIPMAP_LINEAR", 0, OnChange_gl_texturemode };
 cvar_t gl_texturemode_viewmodels = { "gl_texturemode_viewmodels", "GL_LINEAR", 0, OnChange_gl_texturemode };
-cvar_t gl_anisotropy = { "gl_anisotropy","1", 0, OnChange_gl_anisotropy };
+cvar_t gl_anisotropy = { "gl_anisotropy","16", 0, OnChange_gl_anisotropy };
```

Comment:
Think this has close to zero performance impact on hardware not from the stone age, and there is a menu item to control it. With low settings both renderers easily filter out for example "metal grate" fence textures at steep rendering angles.